LICENSE

```diff
@@ -1,21 +1,13 @@
-The MIT License (MIT)
```

Comment:
Do tego pliku powinna pójść chyba cała treść licencji Apache 2.0, a to co tutaj dałeś powinno pójść do pliku `NOTICE`. Przerabiam to aktualnie w svg_optimizer i `pana` przy analizowaniu projektu ucina pub score na `pub.dev` za taką licencję i jak przeglądałem inne projekty z licencją Apache 2.0, to mają dodane tylko ten defaultowy tekst licencji, który zapewnia github.